packages/swing-store/src/importer.js

```diff
@@ -0,0 +1,148 @@
+import { Readable } from 'stream';
+import { Fail, q } from '@agoric/assert';
+
+import { makeSwingStore } from './swingStore.js';
+import { buffer } from './util.js';
+
+export function assertComplete(internal, level) {
```

Comment:
```suggestion
/**
 * @param {import('./swingStore').SwingStoreInternal} internal
 * @param {'operational'} level
 * @returns {void}
 */
export function assertComplete(internal, level) {
```
Also, where do we define the level semantics?

Reply:
we don't, but I'll add it to data-export.md along with https://github.com/Agoric/agoric-sdk/pull/8075#discussion_r1277930487